gui_views/button_view.py

```diff
@@ -252,6 +275,15 @@ def copyConnection(self, selectedObject):
     def undo_button_pressed(self):
         logging.debug("undo button pressed")
 
+    #TODO
+    def redo_button_pressed(self):
+        logging.debug("redo button pressed")
+
+    def zoom(self, val):
```

Comment:
Add a teeny weeny docstring to the fn if u can. Just to make everything consistent for code quality.